Kitodo-API/src/main/java/org/kitodo/api/validation/longtermpreservation/LongTermPreservationValidationInterface.java

```diff
@@ -12,12 +12,10 @@
 package org.kitodo.api.validation.longtermpreservation;
 
 import java.net.URI;
-
-import org.kitodo.api.validation.ValidationInterface;
-import org.kitodo.api.validation.ValidationResult;
+import java.util.List;
 
 /** Validation for long-term preservation. */
-public interface LongTermPreservationValidationInterface extends ValidationInterface {
+public interface LongTermPreservationValidationInterface {
```

Comment:
@solth The fact that `LongTermPreservationValidationInterface` should be an instance of `ValidationInterface` was specified during the development of Production 3.x. I've never been happy with the interface specifications and am happy if we adapt them in a meaningful way, but I'm pointing them out here, and the decision is yours to make.